docs/docs/platform/activity-feed.md

```diff
@@ -4,7 +4,7 @@ sidebar_position: 1
 
 # Activity Feed
 
-Novu provides a notification activity feed that monitors every outgoing message associated with its relevant metadata.
+Novu provides a notification activity feed that monitors every outgoing messages associated with its relevant metadata.
```

Comment:
I actually believe 'message' was the right way here.  Maybe @p-fernandez could also add his grammar input here :)